views/webrtc-view/src/components/UserGrid/UserGrid.module.css

```
@@ -1,6 +1,13 @@
 .grid {
   display: grid;
   grid-template-columns: repeat(var(--grid-col-size), 1fr);
+  grid-auto-rows: 1fr;
   grid-gap: var(--j-space-500);
```

Stylelint (17.3.0)

[error] Expected "grid-gap" to be "gap" (property-no-deprecated)

(property-no-deprecated)